container-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+#
+# Copyright (C) 2023 Olliver Schinagl <oliver@schinagl.nl>
+#
+# A beginning user should be able to docker run image bash (or sh) without
+# needing to learn about --entrypoint
+# https://github.com/docker-library/official-images#consistency
```

Comment:
Missed that: please do not commit personal copyright informations and/or license informations to open source projects. Also the whole commented out section isn't necessary IMHO, the rationale for adding a proper entrypoint was already clear enough in the commit comments.

Reply:
You know that even in the linux kernel this is very normal, it's the 'standard' 'author' if you will line. Even if you do a CLA, you sign over your copyright, but still retain your own. Heck, this even went through cisco's legal department, where the only request was, to also add their copyright (as per CLA).

I'm fine with removing it, I don't care, but this is very standard 'run-of-the-mill' stuff, done pretty much in every opensource project ;)

As for the 'explanation' sure, I don't mind removing it :)

Reply:
No, I did not know, I've never seen it done even once on any project I helped maintain or contributed to over the years (but admittedly they're not on the same level as the linux kernel or Cisco stuff), so I assumed something that might have been wrong.

I don't have strong feeling about it aside from finding it somewhat odd to do it on a repo where this was never ever done by any other contributor. If you rather commit your copyright go ahead.

Reply:
I don't mind, I've used this same template for years now; But it's your repo, your the maintainer, so I'll do what you prefer, well already did :)